idaes/power_generation/unit_models/boiler_heat_exchanger.py

```diff
@@ -77,11 +79,92 @@ class DeltaTMethod(Enum):
     coCurrent = 1
 
 
+def delta_temperature_lmtd_callback(b):
```

Comment:
I didn't do a detailed comparison, but are these the same the ones in the base HX module? If so, shouldn't we just import them from there?

Reply:
Yeah.  I think they are I'll double check and do that.

Reply:
Ok, this exists in the base heat exchanger, but it's named different, so I'll leave it.  Ideally this would inherit the base heatexchanger, but I guess I won't push that.  It's probably too late to go back.

Reply:
Okay.  I figured out why I rewrote the call backs here.  This model has different variable and expression names than the generic heat exchanger.

Reply:
Should we standardize the names between the two models then? To someone who knows nothing about the details of the model, it sounds like that would be a good idea.

Reply:
I feel like that ship has sailed.  We should inherit the standard heat exchanger, but there are a lot of legacy models (not automatically tested and not in the repo) that probably depend on us not changing this in a way that would break backward compatibility.

Reply:
I am not sure how I feel about that. whilst I don't want to break other peoples models, we really should try to enforce some standards on naming, otherwise we are just going to end up proliferating models which exist only because someone used a different set of names. I will also note that whilst we guarantee that if there is a test we won't break it, the reverse is not true.

Is there a way we could set up some `References` or some deprecation path so that we could standardize this?

Reply:
Probably.  I'll take a look.

Reply:
Ok, I added the references and used the callbacks from the generic heat exchanger.